### ISSUE TITLE
Have you tried using the quantized mobilenet from caffe2?

### ISSUE THREAD

**Issue by yxchng** [cedrickchee/pytorch-android#5]
https://github.com/caffe2/models/tree/master/mobilenet_v2_quantized#model

**Comment by cedrickchee** [comment]
I have not tried.

**Comment by cedrickchee** [comment]
I have tried running the quantized MobileNet v2 from your link. According to the docs there, I have ensure my Caffe2 has QNNPACK based Int8 operators before running inference on it.

But, the Android app crashed with the following error:

```
A/native: [F int8_fc_op.h:43] Check failed: K == W.t.size(1) 
    terminating.
A/libc: Fatal signal 6 (SIGABRT), code -6 in tid 7829 (Camera Backgrou), pid 7790 (facebook.f8demo)
```

I fixed that by normalizing the input images in the same way the pre-trained model expect by modifying `native-lib.cpp`. This time, the app didn't crash but stuck at the "LOADING" view.
